server/src/main/java/io/spine/server/entity/AbstractVersionableEntity.java

```diff
@@ -173,6 +174,7 @@ private void updateVersion(Version newVersion) {
      *
      * @param newState a new state to set
      */
+    @VisibleForTesting
```

Comment:
I think this annotation can be removed. Let's discuss if needed, but it shouldn't be here.